module-3/app/service/mythicalMysfitsService.go

```diff
@@ -0,0 +1,49 @@
+package main
+
+import (
+	"fmt"
+	"net/http"
+
+	L "./client"
```

Comment:
Avoid the use of relative imports. Let me know if you are having trouble using fully qualified package imports in your branch and I'll help you work through that. 

Alias of `L` for client package being imported could also be renamed to be more self-documenting

Reply:
I'll take another run at the imports.

Reply:
I'll take another look at the imports.